geographiclib geographiclib/2.3: Library 'GeographicLib' not found in package

### Description

When attempting to use geographiclib/2.3 I get Library 'GeographicLib' not found in package.
This is caused by find_package(geographiclib CONFIG REQUIRED)


### Package and Environment Details

- Conan2
- CMake Conan
- Dev Containers
- Gcc 12
- C++20
- cmake version 3.29.2

### Conan profile

Host profile:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=11
os=Linux

Build profile:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=11
os=Linux

### Steps to reproduce

https://conan.io/center/recipes/geographiclib?version=2.3
Follow steps under here when consuming the library...

>find_package(geographiclib REQUIRED)
target_link_libraries(YOUR_TARGET GeographicLib::GeographicLib)

### Logs

<details><summary>Click to expand log</summary>

```
[main] Building folder: vulkan_triangle 
[build] Starting build
[proc] Executing command: /usr/local/bin/cmake --build /workspaces/vulkan_triangle/builds/vulkan_triangle --config Debug
[build] [0/2] Re-checking globbed directories...
[build] [1/2] Re-running CMake...
[build] CMake Deprecation Warning at builds/vulkan_triangle/CMakeRC.cmake:37 (cmake_minimum_required):
[build]   Compatibility with CMake < 3.5 will be removed from a future version of
[build]   CMake.
[build] 
[build]   Update the VERSION argument <min> value or use a ...<max> suffix to tell
[build]   CMake that the project does not need compatibility with older versions.
[build] Call Stack (most recent call first):
[build]   CMakeLists.txt:14 (include)
[build] 
[build] 
[build] -- CMake-Conan: first find_package() found. Installing dependencies with Conan
[build] -- CMake-Conan: Checking if a default profile exists
[build] /home/spock/.conan2/profiles/default
[build] -- CMake-Conan: cmake_system_name=Linux
[build] -- CMake-Conan: cmake_system_processor=x86_64
[build] -- CMake-Conan: CMake compiler=GNU
[build] -- CMake-Conan: CMake compiler version=12.3.0
[build] -- CMake-Conan: [settings] compiler=gcc
[build] -- CMake-Conan: [settings] compiler.version=12
[build] -- CMake-Conan: Creating profile /workspaces/vulkan_triangle/builds/vulkan_triangle/conan_host_profile
[build] -- CMake-Conan: Profile: 
[build] [settings]
[build] arch=x86_64
[build] os=Linux
[build] compiler=gcc
[build] compiler.version=12
[build] compiler.cppstd=20
[build] compiler.libcxx=libstdc++11
[build] [conf]
[build] tools.cmake.cmaketoolchain:generator=Ninja Multi-Config
[build] tools.build:compiler_executables={"c":"/usr/bin/gcc-12","cpp":"/usr/bin/g++-12"}
[build] 
[build] -- CMake-Conan: Installing both Debug and Release
[build] -- CMake-Conan: conan install /workspaces/vulkan_triangle -of=/workspaces/vulkan_triangle/builds/vulkan_triangle/conan --profile:host=default;--profile:host=/workspaces/vulkan_triangle/builds/vulkan_triangle/conan_host_profile;--profile:build=default;-s;build_type=Release;--build=missing
[build] 
[build] ======== Input profiles ========
[build] Profile host:
[build] [settings]
[build] arch=x86_64
[build] build_type=Release
[build] compiler=gcc
[build] compiler.cppstd=20
[build] compiler.libcxx=libstdc++11
[build] compiler.version=12
[build] os=Linux
[build] [conf]
[build] tools.build:compiler_executables={'c': '/usr/bin/gcc-12', 'cpp': '/usr/bin/g++-12'}
[build] tools.cmake.cmaketoolchain:generator=Ninja Multi-Config
[build] 
[build] Profile build:
[build] [settings]
[build] arch=x86_64
[build] build_type=Release
[build] compiler=gcc
[build] compiler.cppstd=gnu17
[build] compiler.libcxx=libstdc++11
[build] compiler.version=11
[build] os=Linux
[build] 
[build] 
[build] ======== Computing dependency graph ========
[build] Graph root
[build]     conanfile.py: /workspaces/vulkan_triangle/conanfile.py
[build] Requirements
[build]     argparse/3.0#0ea227995ac88102079b03360db5fb78 - Cache
[build]     fmt/10.2.1#9199a7a0611866dea5c8849a77467b25 - Cache
[build]     geographiclib/2.3#13d1f98516c71adcbb2ee6dcce6f98ae - Cache
[build]     glm/cci.20230113#c9c89bea128786a245028f0cb4c1936d - Cache
[build]     gtest/1.14.0#25e2a474b4d1aecf5ff4f0555dcdf72c - Cache
[build]     imgui/1.90.5-docking#377ed750dc6eeed98e9e1647b43d515d - Cache
[build]     libenvpp/1.4.0#0ce210429b9a55a93f18833016cc99be - Cache
[build]     namedtype/20190324#a43a746ef2063d2c3290766967b76cdb - Cache
[build]     perfetto/41.0#0bf135670acf3a691666ba70a7de0c70 - Cache
[build]     spdlog/1.13.0#8e88198fd5b9ee31d329431a6d0ccaa2 - Cache
[build]     stb/cci.20220909#d731359593a5b94567c30fd1df244141 - Cache
[build]     tinyobjloader/2.0.0-rc10#1f75250f5a1b3a0373ce8e134061b318 - Cache
[build]     units/2.3.3#a1ff81d2b51dc794fbf14f9cab755c07 - Cache
[build]     vk-bootstrap/0.7#65edcbf2428a70b711b465435c1c2ebc - Cache
[build]     vulkan-headers/1.3.239.0#4d5b2ede056f66391b085c76105d7fae - Cache
[build] Build requirements
[build]     cmake/3.29.5#56dbff3ff8af6bcd4b9c170fbd4da068 - Cache
[build] Resolved version ranges
[build]     cmake/[>=3.16 <4]: cmake/3.29.5
[build] 
[build] ======== Computing necessary packages ========
[build] Requirements
[build]     argparse/3.0#0ea227995ac88102079b03360db5fb78:da39a3ee5e6b4b0d3255bfef95601890afd80709#1f9b4fda66663f7dc00fdf8accc7af99 - Cache
[build]     fmt/10.2.1#9199a7a0611866dea5c8849a77467b25:346eeea972707b437177bc082df8ccaee893986f#a817f3f95a8162c501eafcdff35a2ddd - Cache
[build]     geographiclib/2.3#13d1f98516c71adcbb2ee6dcce6f98ae:ff975bc68212f19f9a1c28c4b3abb2d3d7d99e40#c1bc95d4dfcb48acb2c32d8fae24690f - Cache
[build]     glm/cci.20230113#c9c89bea128786a245028f0cb4c1936d:da39a3ee5e6b4b0d3255bfef95601890afd80709#6e0f3e7e82ccc211e8cc3c012d8c42ef - Cache
[build]     gtest/1.14.0#25e2a474b4d1aecf5ff4f0555dcdf72c:8d0953ce4a9fbc90d8a791f13de4779da6e14217#643a73ca24a05fa1ceceba5982b050e4 - Cache
[build]     imgui/1.90.5-docking#377ed750dc6eeed98e9e1647b43d515d:8173e7b49ded508895574d119511199292d8a6da#7432d41d54b5e64d4544ce7a80e6b56c - Cache
[build]     libenvpp/1.4.0#0ce210429b9a55a93f18833016cc99be:430b972732f26da8b712fc8a5514cd80cd43cac0#ea72d47c90ad2ccd22efc5bd6b39d50f - Cache
[build]     namedtype/20190324#a43a746ef2063d2c3290766967b76cdb:da39a3ee5e6b4b0d3255bfef95601890afd80709#6ca991738699f8ddfc8599738518e5a8 - Cache
[build]     perfetto/41.0#0bf135670acf3a691666ba70a7de0c70:83e887a5cc8fdb5c4c7b24d6fc4df6d52ac62405#cfdedcb1806d3deb6e2bd1b2a087c13a - Cache
[build]     spdlog/1.13.0#8e88198fd5b9ee31d329431a6d0ccaa2:f5b7aa9f4a755cf291d49850f74a9848023def62#c74a5deb35acb327e3f73485c06d01d0 - Cache
[build]     stb/cci.20220909#d731359593a5b94567c30fd1df244141:24b381c7532f70c284a2a67cc83f779b3c0042fb#e100cac4b7018171ddb0bdd183f286d1 - Cache
[build]     tinyobjloader/2.0.0-rc10#1f75250f5a1b3a0373ce8e134061b318:283c9608e6f2261ed42e5d1df932bd7b0682641f#875d601e2ca3dc718a5127a6ef24923b - Cache
[build]     units/2.3.3#a1ff81d2b51dc794fbf14f9cab755c07:da39a3ee5e6b4b0d3255bfef95601890afd80709#b2dbcf0af7a99cdfb51e8cba06504e9e - Cache
[build]     vk-bootstrap/0.7#65edcbf2428a70b711b465435c1c2ebc:c2628eec1f940a98f7a18d54b9c9f2432dfe3e95#ddf47f1eef2a62c2fc525e453c2d91ec - Cache
[build]     vulkan-headers/1.3.239.0#4d5b2ede056f66391b085c76105d7fae:da39a3ee5e6b4b0d3255bfef95601890afd80709#854aca8847e06c517b5861c97603fe0c - Cache
[build] Build requirements
[build] Skipped binaries
[build]     cmake/3.29.5
[build] 
[build] ======== Installing packages ========
[build] argparse/3.0: Already installed! (1 of 15)
[build] fmt/10.2.1: Already installed! (2 of 15)
[build] geographiclib/2.3: Already installed! (3 of 15)
[build] glm/cci.20230113: Already installed! (4 of 15)
[build] gtest/1.14.0: Already installed! (5 of 15)
[build] imgui/1.90.5-docking: Already installed! (6 of 15)
[build] imgui/1.90.5-docking: Appending PATH env var with : /home/spock/.conan2/p/b/imgui5b9cc270e58b7/p/bin
[build] namedtype/20190324: Already installed! (7 of 15)
[build] perfetto/41.0: Already installed! (8 of 15)
[build] stb/cci.20220909: Already installed! (9 of 15)
[build] tinyobjloader/2.0.0-rc10: Already installed! (10 of 15)
[build] units/2.3.3: Already installed! (11 of 15)
[build] vulkan-headers/1.3.239.0: Already installed! (12 of 15)
[build] libenvpp/1.4.0: Already installed! (13 of 15)
[build] spdlog/1.13.0: Already installed! (14 of 15)
[build] vk-bootstrap/0.7: Already installed! (15 of 15)
[build] WARN: deprecated: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
[build] WARN: deprecated:     'cpp_info.names' used in: geographiclib/2.3, gtest/1.14.0, spdlog/1.13.0, fmt/10.2.1, vulkan-headers/1.3.239.0
[build] WARN: deprecated:     'cpp_info.filenames' used in: geographiclib/2.3, vulkan-headers/1.3.239.0
[build] WARN: deprecated:     'env_info' used in: geographiclib/2.3, imgui/1.90.5-docking
[build] WARN: deprecated:     'cpp_info.build_modules' used in: tinyobjloader/2.0.0-rc10
[build] 
[build] ======== Finalizing install (deploy, generators) ========
[build] conanfile.py: Writing generators to /workspaces/vulkan_triangle/builds/vulkan_triangle/conan
[build] conanfile.py: Generator 'CMakeDeps' calling 'generate()'
[build] conanfile.py: CMakeDeps necessary find_package() and targets for your CMakeLists.txt
[build]     find_package(spdlog)
[build]     find_package(vk-bootstrap)
[build]     find_package(glm)
[build]     find_package(stb)
[build]     find_package(imgui)
[build]     find_package(tinyobjloader)
[build]     find_package(geographiclib)
[build]     find_package(perfetto)
[build]     find_package(argparse)
[build]     find_package(GTest)
[build]     find_package(libenvpp)
[build]     find_package(units)
[build]     find_package(namedtype)
[build]     target_link_libraries(... spdlog::spdlog vk-bootstrap::vk-bootstrap glm::glm stb::stb imgui::imgui tinyobjloader::tinyobjloader GeographicLib::GeographicLib perfetto::perfetto argparse::argparse gtest::gtest libenvpp::libenvpp units::units namedtype::namedtype)
[build] conanfile.py: Generating aggregated env files
[build] conanfile.py: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
[build] Install finished successfully
[build] -- CMake-Conan: CONAN_GENERATORS_FOLDER=/workspaces/vulkan_triangle/builds/vulkan_triangle/conan
[build] -- CMake-Conan: CONANFILE=/workspaces/vulkan_triangle/conanfile.py
[build] -- CMake-Conan: conan install /workspaces/vulkan_triangle -of=/workspaces/vulkan_triangle/builds/vulkan_triangle/conan --profile:host=default;--profile:host=/workspaces/vulkan_triangle/builds/vulkan_triangle/conan_host_profile;--profile:build=default;-s;build_type=Debug;--build=missing
[build] 
[build] ======== Input profiles ========
[build] Profile host:
[build] [settings]
[build] arch=x86_64
[build] build_type=Debug
[build] compiler=gcc
[build] compiler.cppstd=20
[build] compiler.libcxx=libstdc++11
[build] compiler.version=12
[build] os=Linux
[build] [conf]
[build] tools.build:compiler_executables={'c': '/usr/bin/gcc-12', 'cpp': '/usr/bin/g++-12'}
[build] tools.cmake.cmaketoolchain:generator=Ninja Multi-Config
[build] 
[build] Profile build:
[build] [settings]
[build] arch=x86_64
[build] build_type=Release
[build] compiler=gcc
[build] compiler.cppstd=gnu17
[build] compiler.libcxx=libstdc++11
[build] compiler.version=11
[build] os=Linux
[build] 
[build] 
[build] ======== Computing dependency graph ========
[build] Graph root
[build]     conanfile.py: /workspaces/vulkan_triangle/conanfile.py
[build] Requirements
[build]     argparse/3.0#0ea227995ac88102079b03360db5fb78 - Cache
[build]     fmt/10.2.1#9199a7a0611866dea5c8849a77467b25 - Cache
[build]     geographiclib/2.3#13d1f98516c71adcbb2ee6dcce6f98ae - Cache
[build]     glm/cci.20230113#c9c89bea128786a245028f0cb4c1936d - Cache
[build]     gtest/1.14.0#25e2a474b4d1aecf5ff4f0555dcdf72c - Cache
[build]     imgui/1.90.5-docking#377ed750dc6eeed98e9e1647b43d515d - Cache
[build]     libenvpp/1.4.0#0ce210429b9a55a93f18833016cc99be - Cache
[build]     namedtype/20190324#a43a746ef2063d2c3290766967b76cdb - Cache
[build]     perfetto/41.0#0bf135670acf3a691666ba70a7de0c70 - Cache
[build]     spdlog/1.13.0#8e88198fd5b9ee31d329431a6d0ccaa2 - Cache
[build]     stb/cci.20220909#d731359593a5b94567c30fd1df244141 - Cache
[build]     tinyobjloader/2.0.0-rc10#1f75250f5a1b3a0373ce8e134061b318 - Cache
[build]     units/2.3.3#a1ff81d2b51dc794fbf14f9cab755c07 - Cache
[build]     vk-bootstrap/0.7#65edcbf2428a70b711b465435c1c2ebc - Cache
[build]     vulkan-headers/1.3.239.0#4d5b2ede056f66391b085c76105d7fae - Cache
[build] Build requirements
[build]     cmake/3.29.5#56dbff3ff8af6bcd4b9c170fbd4da068 - Cache
[build] Resolved version ranges
[build]     cmake/[>=3.16 <4]: cmake/3.29.5
[build] 
[build] ======== Computing necessary packages ========
[build] Requirements
[build]     argparse/3.0#0ea227995ac88102079b03360db5fb78:da39a3ee5e6b4b0d3255bfef95601890afd80709#1f9b4fda66663f7dc00fdf8accc7af99 - Cache
[build]     fmt/10.2.1#9199a7a0611866dea5c8849a77467b25:9ba98d78b757b902def4d230783d0f128f557a1f#94e52beaba2c3300aab965157ada5192 - Cache
[build]     geographiclib/2.3#13d1f98516c71adcbb2ee6dcce6f98ae:647be41f3c92bfb1857e3a6bee963d873528cb09#9b06ca9d5c3ccc381f096cf57c9e1d12 - Cache
[build]     glm/cci.20230113#c9c89bea128786a245028f0cb4c1936d:da39a3ee5e6b4b0d3255bfef95601890afd80709#6e0f3e7e82ccc211e8cc3c012d8c42ef - Cache
[build]     gtest/1.14.0#25e2a474b4d1aecf5ff4f0555dcdf72c:4e4507935723f635134b24131c8bf53db33bb8be#4a3cfbb299c8e1c0055b69bbe2dd09ee - Cache
[build]     imgui/1.90.5-docking#377ed750dc6eeed98e9e1647b43d515d:7cc5800710ac64d28a0166f497ea6fa6c6bc3a60#c6b2757f0cff06eda602bef5aa71b77a - Cache
[build]     libenvpp/1.4.0#0ce210429b9a55a93f18833016cc99be:dcf110068e4d3219b4830a176962dcd5e2cab87b#13697b15c5a4e144829a7b12de31fdde - Cache
[build]     namedtype/20190324#a43a746ef2063d2c3290766967b76cdb:da39a3ee5e6b4b0d3255bfef95601890afd80709#6ca991738699f8ddfc8599738518e5a8 - Cache
[build]     perfetto/41.0#0bf135670acf3a691666ba70a7de0c70:378899a52ef8214b94dbbe58715bb5e2d1e054ef#d76cdf1d916ab9b8aab6ab07648e367e - Cache
[build]     spdlog/1.13.0#8e88198fd5b9ee31d329431a6d0ccaa2:41f9a55ff5579f90f99d383c24486a361a1e4aea#b44b3703d6bfa11095629ee152006300 - Cache
[build]     stb/cci.20220909#d731359593a5b94567c30fd1df244141:24b381c7532f70c284a2a67cc83f779b3c0042fb#e100cac4b7018171ddb0bdd183f286d1 - Cache
[build]     tinyobjloader/2.0.0-rc10#1f75250f5a1b3a0373ce8e134061b318:2252bafe5f97e3ce3c355c89fd4e53602a4f977c#43d77620c7897cf5fa4ec60117d05588 - Cache
[build]     units/2.3.3#a1ff81d2b51dc794fbf14f9cab755c07:da39a3ee5e6b4b0d3255bfef95601890afd80709#b2dbcf0af7a99cdfb51e8cba06504e9e - Cache
[build]     vk-bootstrap/0.7#65edcbf2428a70b711b465435c1c2ebc:7e06d6d49eaad777c023051d96257b038f02b3ab#5359eae3534a74da7e44eb1008a316f7 - Cache
[build]     vulkan-headers/1.3.239.0#4d5b2ede056f66391b085c76105d7fae:da39a3ee5e6b4b0d3255bfef95601890afd80709#854aca8847e06c517b5861c97603fe0c - Cache
[build] Build requirements
[build] Skipped binaries
[build]     cmake/3.29.5
[build] 
[build] ======== Installing packages ========
[build] argparse/3.0: Already installed! (1 of 15)
[build] fmt/10.2.1: Already installed! (2 of 15)
[build] geographiclib/2.3: Already installed! (3 of 15)
[build] glm/cci.20230113: Already installed! (4 of 15)
[build] gtest/1.14.0: Already installed! (5 of 15)
[build] imgui/1.90.5-docking: Already installed! (6 of 15)
[build] imgui/1.90.5-docking: Appending PATH env var with : /home/spock/.conan2/p/b/imgui43b17b5f91e6f/p/bin
[build] namedtype/20190324: Already installed! (7 of 15)
[build] perfetto/41.0: Already installed! (8 of 15)
[build] stb/cci.20220909: Already installed! (9 of 15)
[build] tinyobjloader/2.0.0-rc10: Already installed! (10 of 15)
[build] units/2.3.3: Already installed! (11 of 15)
[build] vulkan-headers/1.3.239.0: Already installed! (12 of 15)
[build] libenvpp/1.4.0: Already installed! (13 of 15)
[build] spdlog/1.13.0: Already installed! (14 of 15)
[build] vk-bootstrap/0.7: Already installed! (15 of 15)
[build] WARN: deprecated: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
[build] WARN: deprecated:     'cpp_info.names' used in: geographiclib/2.3, spdlog/1.13.0, fmt/10.2.1, vulkan-headers/1.3.239.0, gtest/1.14.0
[build] WARN: deprecated:     'cpp_info.filenames' used in: geographiclib/2.3, vulkan-headers/1.3.239.0
[build] WARN: deprecated:     'env_info' used in: imgui/1.90.5-docking, geographiclib/2.3
[build] WARN: deprecated:     'cpp_info.build_modules' used in: tinyobjloader/2.0.0-rc10
[build] 
[build] ======== Finalizing install (deploy, generators) ========
[build] conanfile.py: Writing generators to /workspaces/vulkan_triangle/builds/vulkan_triangle/conan
[build] conanfile.py: Generator 'CMakeDeps' calling 'generate()'
[build] conanfile.py: CMakeDeps necessary find_package() and targets for your CMakeLists.txt
[build]     find_package(spdlog)
[build]     find_package(vk-bootstrap)
[build]     find_package(glm)
[build]     find_package(stb)
[build]     find_package(imgui)
[build]     find_package(tinyobjloader)
[build]     find_package(geographiclib)
[build]     find_package(perfetto)
[build]     find_package(argparse)
[build]     find_package(GTest)
[build]     find_package(libenvpp)
[build]     find_package(units)
[build]     find_package(namedtype)
[build]     target_link_libraries(... spdlog::spdlog vk-bootstrap::vk-bootstrap glm::glm stb::stb imgui::imgui tinyobjloader::tinyobjloader GeographicLib::GeographicLib perfetto::perfetto argparse::argparse gtest::gtest libenvpp::libenvpp units::units namedtype::namedtype)
[build] conanfile.py: Generating aggregated env files
[build] conanfile.py: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
[build] Install finished successfully
[build] -- CMake-Conan: CONAN_GENERATORS_FOLDER=/workspaces/vulkan_triangle/builds/vulkan_triangle/conan
[build] -- CMake-Conan: CONANFILE=/workspaces/vulkan_triangle/conanfile.py
[build] -- Could NOT find Vulkan (missing: Vulkan_DIR)
[build] -- CMake-Conan: find_package(spdlog) found, 'conan install' already ran
[build] -- Conan: Component target declared 'spdlog::spdlog'
[build] -- CMake-Conan: find_package(fmt) found, 'conan install' already ran
[build] -- Conan: Component target declared 'fmt::fmt'
[build] -- CMake-Conan: find_package(VulkanHeaders) found, 'conan install' already ran
[build] -- Conan: Component target declared 'Vulkan::Registry'
[build] -- Conan: Target declared 'vulkan-headers::vulkan-headers'
[build] -- CMake-Conan: find_package(glfw3) found, 'conan install' already ran
[build] -- CMake-Conan: find_package(vk-bootstrap) found, 'conan install' already ran
[build] -- Conan: Target declared 'vk-bootstrap::vk-bootstrap'
[build] -- CMake-Conan: find_package(glm) found, 'conan install' already ran
[build] -- Conan: Target declared 'glm::glm'
[build] -- CMake-Conan: find_package(stb) found, 'conan install' already ran
[build] -- Conan: Target declared 'stb::stb'
[build] -- CMake-Conan: find_package(imgui) found, 'conan install' already ran
[build] -- Conan: Target declared 'imgui::imgui'
[build] -- CMake-Conan: find_package(tinyobjloader) found, 'conan install' already ran
[build] -- Conan: Target declared 'tinyobjloader::tinyobjloader'
[build] -- CMake-Conan: find_package(geographiclib) found, 'conan install' already ran
[build] -- Conan: Target declared 'GeographicLib::GeographicLib'
[build] CMake Error at builds/vulkan_triangle/conan/cmakedeps_macros.cmake:67 (message):
[build]   Library 'GeographicLib' not found in package.  If 'GeographicLib' is a
[build]   system library, declare it with 'cpp_info.system_libs' property
[build] Call Stack (most recent call first):
[build]   builds/vulkan_triangle/conan/geographiclib-Target-debug.cmake:23 (conan_package_library_targets)
[build]   builds/vulkan_triangle/conan/geographiclibTargets.cmake:24 (include)
[build]   builds/vulkan_triangle/conan/geographiclib-config.cmake:16 (include)
[build]   cmake/conan_provider.cmake:594 (find_package)
[build]   CMakeLists.txt:28 (find_package)
[build] 
[build] 
[build] -- Configuring incomplete, errors occurred!
[build] FAILED: CMakeFiles/impl-Debug.ninja build-Debug.ninja CMakeFiles/impl-Release.ninja build-Release.ninja build.ninja 
[build] /usr/local/bin/cmake --regenerate-during-build -S/workspaces/vulkan_triangle -B/workspaces/vulkan_triangle/builds/vulkan_triangle
[build] ninja: error: rebuilding 'build-Debug.ninja': subcommand failed
[proc] The command: /usr/local/bin/cmake --build /workspaces/vulkan_triangle/builds/vulkan_triangle --config Debug exited with code: 1
[driver] Build completed: 00:00:01.558
[build] Build finished with exit code 1

```

</details>

--- a/recipes/geographiclib/all/conanfile.py
+++ b/recipes/geographiclib/all/conanfile.py
@@ -4,7 +4,7 @@ from conan.tools.build import check_min_cppstd
 from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 from conan.tools.files import (
     apply_conandata_patches, copy, export_conandata_patches, get,
-    replace_in_file, rm, rmdir
+    replace_in_file, rm, rmdir, collect_libs
 )
 from conan.tools.scm import Version
 from conan.tools.microsoft import is_msvc
@@ -138,9 +138,11 @@ class GeographiclibConan(ConanFile):
         self.cpp_info.set_property("cmake_file_name", "geographiclib")
         self.cpp_info.set_property("cmake_target_name", "GeographicLib::GeographicLib")
         self.cpp_info.set_property("pkg_config_name", "geographiclib")
-        suffix = "_d" if self.settings.build_type == "Debug" and is_msvc(self) else ""
-        suffix += "-i" if is_msvc(self) and self.options.shared else ""
-        self.cpp_info.libs = [f"GeographicLib{suffix}"] if Version(self.version) >= "2" else [f"Geographic{suffix}"]
+        # Geographic library name is GeographicLib since version 2.x (was Geographic before)
+        # It uses a debug postfix _d on Windows or when using multi-configuration generators (like Ninja Multi-Config)
+        # It's hard to track when using multi-configuration generators, so collect_libs is used
+        # Plus, it adds -i postfix on Windows when using shared libraries
+        self.cpp_info.libs = collect_libs(self)
         self.cpp_info.defines.append("GEOGRAPHICLIB_SHARED_LIB={}".format("1" if self.options.shared else "0"))
         if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.system_libs = ["m"]

